sys/include/net/ng_netreg.h

```diff
@@ -75,6 +75,9 @@ void ng_netreg_init(void);
  * @param[in] entry     An entry you want to add to the registry with
  *                      ng_netreg_entry_t::pid and ng_netreg_entry_t::demux_ctx set.
  *
+ * @warning Call ng_netreg_unregister() *before* you leave the context you
+ *          allocated @p entry in. Otherwise it might get overwritten.
+ *
```

Comment:
But in the hackathon it wasn't clear for non-RIOTers that calling `ng_netreg_register()` on a stack-allocated object is not a good idea, if you don't unregister it before popping the current stack frame.